File: 28/python/main.py
#!/usr/bin/env python
# coding: -utf8

if __name__ == '__main__':
    total = 0
    for i in range(5):
        user_input = input("Enter a number: ")
        num = int(user_input)
        total += num

    print(f"The total is {total}.\n", end='')

File: 28/cpp/main.cc
#include <iostream>

namespace {
std::string input(const std::string& msg) {
    std::string user_input;
    std::cout << msg;
    std::getline(std::cin, user_input);
    return user_input;
}
}

int main() {
    int total = 0;
    for (int i = 0; i < 5; i++) {
        auto user_input = input("Enter a number: ");
        total += std::stoi(user_input);
    }

    std::cout << "The total is " << total << ".\n";

    return 0;
}
